conda/recipes/legate-dataframe/meta.yaml

```diff
@@ -60,6 +60,7 @@ requirements:
     - cudf {{ rapids_version }}  # pulls in pyarrow?
     - libcudf {{ rapids_version }}
     - librmm {{ rapids_version }}
+    - nccl {{ nccl_version }}
```

Comment:
Yeah this is a good fix! The existence of direct references like this:

https://github.com/rapidsai/legate-dataframe/blob/49cc2b9fe586e1144208bad7d7ceddfde09ac6d5/cpp/include/legate_dataframe/core/nccl.hpp#L21

mean we should have a `host:` dependency on `nccl` here.